save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -179,10 +179,18 @@ class VulnerabilityService(
 
         val userIds = vulnerabilities.map { it.userId }
         val userInfos = userRepository.findAllById(userIds)
-        return vulnerabilities.map { vulnerability ->
-            val correspondingUserInfo = userInfos.find { user -> user.id == vulnerability.userId }?.toUserInfo()
-            vulnerability.toDto().copy(userInfo = correspondingUserInfo ?: UserInfo(name = "?"))
-        }
+        val tagMap = lnkVulnerabilityTagRepository.findByVulnerabilityIdIn(vulnerabilities.map { it.requiredId() })
```

Comment:
very complicated. What is the idea? You just need to get all tags by vuln, no?

Reply:
I do, but as far as we have `LnkVulnerabilityTag` and we also want to make one database request instead of `N`, it seems to be the easiest solution